modules/installation-aws-permissions.adoc

```diff
@@ -152,6 +152,11 @@ If you use an existing Virtual Private Cloud (VPC), your account does not requir
 * `iam:TagInstanceProfile`
 * `iam:TagRole`
 
+[NOTE]
+=====
+If you use the bring your own (BYO) `iamRole`, the following IAM permissions are not required for installation: `iam:CreateRole`,`iam:DeleteRole`, `iam:DeleteRolePolicy`, and `iam:PutRolePolicy`.
```

Comment:
`iam:DeleteRole` and `iam:DeleteRolePolicy` is used by the destroy cluster, so I'd suggest update the description like:
```suggestion
If you use the bring your own (BYO) `iamRole`, the following IAM permissions are not required: `iam:CreateRole`,`iam:DeleteRole`, `iam:DeleteRolePolicy`, and `iam:PutRolePolicy`.
```